docs/index.rst

```diff
@@ -0,0 +1,36 @@
+#########################
+Fedora Møte documentation
+#########################
+
+This is the main page for the Fedora Møte documentation.
+Other pages are available for you to navigate and explore below.
+For more information, see the `repo on GitHub <https://github.com/fedora-infra/mote>`_.
+
+.. toctree::
+    :maxdepth: 2
+    :name: dev
+    :caption: Developer Documentation:
+    :glob:
+
+    dev/*
+
+.. toctree::
+    :maxdepth: 2
+    :name: about
+    :caption: About Møte:
```

Comment:
I haven't worked with RSTs so I don't know if the following error message is worth being bothered about or not. :sweat_smile: 

![image](https://user-images.githubusercontent.com/49605954/97329223-d73d0f80-189c-11eb-945d-1a546c9239d6.png)

Reply:
`toctree` is a Sphinx directive, not a part of ReStructuredText. In order for the document to render as expected, the doc needs to be edited in an environment that has Sphinx in the `PATH`. My workflow is usually writing the docs and then rebuilding the docs site locally when I am ready to preview changes.